src/react/queries.ts

```diff
@@ -59,3 +62,50 @@ export const workflowListQuery = (
     enabled: mgnt != null,
   };
 };
+
+export const getUserAccessKeysQuery = (userName: string, IAMClient: IAMClient) => ({
+  queryKey: ['listIAMUserAccessKey', userName],
+  queryFn: (_ctx: QueryFunctionContext, marker: string) => IAMClient.listAccessKeys(userName, marker),
+  enabled: IAMClient !== null,
+  refetchOnMount: false,
+  refetchOnWindowFocus: false
+});
+
+export const getUserListGroupsQuery = (userName: string, IAMClient: IAMClient) => ({
+  queryKey: ['listIAMUserGroups', userName],
+  queryFn: () => IAMClient.listGroupsForUser(userName),
+  enabled: IAMClient !== null,
+});
+
+export const getUserListUsersQuery = (accountName: string, IAMClient: IAMClient) => ({
+  queryKey: ['listIAMUsers', accountName],
+  queryFn: (maxItems: number, marker: string) => IAMClient.listUsers(1000, marker),
+  staleTime: Infinity,
+  enabled: IAMClient !== null,
+  refetchOnMount: false,
+  refetchOnWindowFocus: false,
+})
+
+export const useUserAccessKeysQuery = (userName: string, IAMClient: IAMClient) => {
+  const {
+    data: accessKeysResult,
+    status: userAccessKeyStatus,
+    firstPageStatus: firstPageUserAccessKeyStatus,
+  } = useAwsPaginatedEntities(
+    getUserAccessKeysQuery(userName, IAMClient),
+    data => data.AccessKeyMetadata,
+  );
+  return { firstPageUserAccessKeyStatus, accessKeysResult, userAccessKeyStatus };
+};
+
+export const useUserListGroupsQuery = (userName: string, IAMClient: IAMClient) => {
+  const {
+    data: listGroupsResult,
+    status: listGroupsStatus,
+    firstPageStatus: firstPageUserAccessKeyStatus,
+  } = useAwsPaginatedEntities(
+    getUserListGroupsQuery(userName, IAMClient),
+    data => data.Groups,
+  );
+  return { firstPageUserAccessKeyStatus, listGroupsResult, listGroupsStatus };
+};
```

Comment:
These two methods doesn't seem to be used anywhere

Reply:
They are used in this PR https://github.com/scality/zenko-ui/pull/284
And I didn't delete them after splitting the original PR
So should I remove them?